docs/hardware.md

```diff
@@ -5,44 +5,30 @@
 | [Adafruit Feather M4 Express](https://www.adafruit.com/product/3857) | Atmel SAMD51 (Cortex M4F) | CircuitPython | An economical solution for basic USB keyboards |
 | [Adafruit ItsyBitsy M4 Express](https://www.adafruit.com/product/3800) | Atmel SAMD51 (Cortex M4F) | CircuitPython | A smaller solution for basic USB keyboards |
 
-## Community Supported
-| Board | Chipset | Python Platform | Maintainer | Notes |
-| ----- | ------- | --------------- | ----- | ----- |
-| [pyboard v1.1](https://www.adafruit.com/product/2390) | STM32F405RG (Cortex M4F) | MicroPython | kdb424 | Very large and expensive, and has ram limitations. |
 
 ## Support Planned/WIP
 | Board | Chipset | Python Platform | Notes |
 | ----- | ------- | --------------- | ----- |
 | [Seeed nRF52840 Micro Dev Kit](https://www.seeedstudio.com/nRF52840-Micro-Development-Kit-p-3079.html) | nRF52840 | [CircuitPython](https://github.com/KMKfw/circuitpython/tree/topic-nrf52840-mdk) | This is basically as bleeding edge as it gets. Will support BLE HID to PC as well as BLE split boards |
```

Comment:
I guess below this we can finally add the [Adafruit Feather nRF52840 Express](https://www.adafruit.com/product/4062) (which is the same chip as the MakerDiary board)

Also this MakerDiary/Seeed board is supported in upstream CircuitPython as of alpha2 or alpha3ish, [not that I'd know](https://github.com/adafruit/circuitpython/pull/1286)

Reply:
Added support, changed wording. Let me know if that's good or resolve please.